src/styles/blocks/catalog.scss

```diff
@@ -12,11 +14,18 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $transition-time, color $transition-time;
```

Comment:
The color transition should be applied to specific elements like the card title and navigation link, not to the whole card. Move the color transition to the appropriate elements.